### ISSUE TITLE
Website stylesheet has larger font size for h3 then h2

### ISSUE THREAD

**Issue by mit-mit** [flutter/flutter#6416]
## Steps to Reproduce

Visit https://flutter.io/debugging/

Notice how `debugger() statement` is a sub-topic under the `Dart Observatory` topic, yet it has a larger font size.

![screen shot 2016-10-20 at 12 42 34](https://cloud.githubusercontent.com/assets/13644170/19556705/9d139c26-96c2-11e6-84bc-690f7d0ce72c.png)

**Comment by mit-mit** [comment]
@sethladd this makes the new IntelliJ docs I am writing really hard to parse. Who knows the details of the stylesheet for the website?

**Comment by sethladd** [comment]
Any of us. None of us. :) We should fix this. Patches welcome if you beat
us to it.

**Comment by devoncarew** [comment]
@qchong - I know you updated the style sheets a while ago - is this something you could help out with?

**Comment by sethladd** [comment]
Thanks!
